Package/choco/IronPython.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>ironpython</id>
+    <version>2.7.8</version>
```

Comment:
Change version?

Reply:
Yes, definitely will when we're ready for releases

Reply:
Oh OK, I saw in the other cases it already says 3.4.0.